### PR TITLE
remove unused imports + changed table size

### PR DESCRIPTION
**Pull request by Siddharthj2002**
NA

### CODE REVIEW

File: src/com/wip/sid/TableDemo.java [hunk 1]
```diff
@@ -204,7 +203,7 @@ public void valueChanged(ListSelectionEvent e) {
 
     public static void main(String[] args) {
         TableDemo td = new TableDemo();
```
Comment (sumitkc) [inline src/com/wip/sid/TableDemo.java]: Revert this change.

Comment (Siddharthj2002) [reply src/com/wip/sid/TableDemo.java]: Reverted

**Review by sumitkc — APPROVED** [review]
Looks good.
